crates/client/src/serialize/txt/parse_rdata.rs

```diff
@@ -22,12 +22,40 @@ use crate::proto::rr::dnssec::rdata::DNSSECRData;
 use crate::rr::{Name, RData, RecordType};
 use crate::serialize::txt::rdata_parsers::*;
 
-pub(crate) trait RDataParser: Sized {
+use crate::serialize::txt::zone_lex::Lexer;
```

Comment:
Feels like this `use` and the one below should be grouped with the block above?

Reply:
Hm. That might have been a mistake. No worries on the timeline. I’ll post a patch to follow this up. I just needed to get these things bound together for some other testing. It was getting unwieldy keeping all the changes straight.